src/components/ClueSelector.js

```diff
@@ -67,7 +67,32 @@ export default function ClueSelector({ board }) {
 
   }, [clueColor]);
 
-  const clues = {red: redClues, blue: blueClues}[clueColor] || [];
+  const getClueString = (n, clues) =>{
```

Comment:
You don't really need to pass `clues` into this function, rather you can use `clues` as if it was a global variable.